biz_portal/apps/portal/templates/portal/business.html

```diff
@@ -1,5 +1,134 @@
+          </a>
+          <a class="mdc-list-item" href="#">
+            <i class="material-icons mdc-list-item__graphic" aria-hidden="true">store</i>
+            <span class="mdc-list-item__text">Add my business</span>
+          </a>
+          <a class="mdc-list-item" href="#">
+            <i class="material-icons mdc-list-item__graphic" aria-hidden="true">arrow_downward</i>
+            <span class="mdc-list-item__text">Download data</span>
+          </a>
+          <a class="mdc-list-item" href="#">
+            <i class="material-icons mdc-list-item__graphic" aria-hidden="true">call</i>
+            <span class="mdc-list-item__text">Contact municipality</span>
+          </a>
+        </nav>
+      </div>
+      </aside>
+      <header class="navbar mdc-top-app-bar mdc-top-app-bar--fixed" id="app-bar">
+        <div class="mdc-top-app-bar__row">
+          <section class="mdc-top-app-bar__section mdc-top-app-bar__section--align-end">
+            <button class="demo-menu material-icons mdc-top-app-bar__navigation-icon">menu</button>
+          </section>
+        </div>
+      </header>
+      <main class="main-content" id="main-content">
+        <div class="mdc-top-app-bar--fixed-adjust top-bar">
+          <div class="business-details-section">
+            <div class="layout">
+              <div>
+                <div class="business-category">
+                    <span class="business-category-text">{{ object.category }}</span>
+                    <span>
+                      <img src="../../../../static/images/accomodation.svg" />
+                    </span>
+                </div>
+                <div class="business-name">{{ object.registered_name }}</div>
+              </div>
+              <div class="business-profile-cards">
+                <details class="mdc-expansion mdc-theme--background mdc-elevation--z2">
+                  <summary class="mdc-expansion__summary">
+                    <span class="business-profile-location mdc-list-item">
+                      <i class="material-icons mdc-list-item__graphic" aria-hidden="true">location_on</i>
+                      <span class="mdc-expansion__header">{{ object.physical_address }}</span>
+                    </span>
+                    <i class="material-icons mdc-list-item__graphic arrow" aria-hidden="true">keyboard_arrow_down</i>
+                  </summary>
+                  <span class="mdc-list-item">
+                    <span class="mdc-expansion__header">{{ object.physical_address }}</span>
+                  </span>
+                </details>
+                <details class="mdc-theme--background mdc-elevation--z2">
+                  <summary class="mdc-expansion__summary">
+                    <span class="business-profile-location mdc-list-item">
+                      <span class="status-icon-span">
+                        <img class="status-icon" src="../../../../static/images/ellipse.svg" />
```

Comment:
this should be in `portal/static` then we can use `{% static 'portal/static/... %}`

Reply:
Covered